requirements.txt

```diff
@@ -1,12 +1,12 @@
 jupyter
-numpy==1.17.4
-pandas==0.25.3
-networkx==2.4
-fire==0.2.1
-giotto-learn==0.1.3
+numpy>=1.17.4
+pandas>=0.25.3
+networkx>=2.4
+fire>=0.2.1
+giotto-tda>=0.1.4
 seaborn==0.9.0
-matplotlib==3.1.0
+matplotlib>=3.1.0
 plotly==4.4.1
-scikit-learn==0.21.3
-xgboost==0.90
-sympy==1.5
+scikit-learn>=0.21.3
```

Comment:
I'd go for >= 0.22.0 as it is what we require from giotto-tda. In fact, I'd even exclude numpy as scikit-learn as these are implied by giotto-tda and they're not going anywhere as dependencies